.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
```

Comment:
Could we move this point below some of the others, as it's not the main point/message we're trying to send?